src/SWP/Bundle/CoreBundle/Controller/ContainerController.php

```diff
@@ -53,7 +53,7 @@ public function listAction(Request $request)
         $entityManager = $this->get('doctrine')->getManager();
         $paginator = $this->get('knp_paginator');
         $containers = $paginator->paginate(
-            $entityManager->getRepository('SWP\Bundle\TemplateEngineBundle\Model\Container')->getAll(),
+            $entityManager->getRepository('SWP\Bundle\CoreBundle\Model\Container')->getAll(),
```

Comment:
you can get repository for container: `swp.repository.container` and widget `swp.repository.widget_model` from Symfony container and also use repositories to persist new objects instead of using entity manager strictly